src/table/table.scss

```diff
@@ -367,6 +411,33 @@
     flex-basis: $iui-l * 2;
   }
 
+  &.iui-sticky {
+    position: sticky;
+    z-index: 1;
```

Comment:
Maybe add `isolation: isolate` somewhere so this doesn't leak out.

Reply:
Where would you suggest on cell or whole table?

Reply:
I think table would make sense because we also have some other other z-indices, like in resizer

Reply:
Added